src/java/picard/sam/RevertSam.java

```diff
@@ -374,6 +374,7 @@ public void revertSamRecord(final SAMRecord rec) {
             rec.setAlignmentStart(SAMRecord.NO_ALIGNMENT_START);
             rec.setCigarString(SAMRecord.NO_ALIGNMENT_CIGAR);
             rec.setMappingQuality(SAMRecord.NO_MAPPING_QUALITY);
+            rec.setAttribute(SAMTag.AS.name(), null);
```

Comment:
What about other alignment based tags, such as ```MD```, ```MC```, or ```NM```?   Why not just add ```AS``` to ```ATTRIBUTE_TO_CLEAR```?